subsys/bluetooth/controller/ll_sw/ull_llcp_cc.c

```diff
@@ -838,13 +838,33 @@ static void lp_cc_st_wait_tx_cis_req(struct ll_conn *conn, struct proc_ctx *ctx,
 	}
 }
 
+static void lp_cc_complete(struct ll_conn *conn, struct proc_ctx *ctx, uint8_t evt, void *param)
+{
+	if (!llcp_ntf_alloc_is_available()) {
+		ctx->state = LP_CC_STATE_WAIT_NTF;
+	} else {
+		cc_ntf_established(conn, ctx);
+		llcp_lr_complete(conn);
+		ctx->state = LP_CC_STATE_IDLE;
+	}
+}
+
 static void lp_cc_st_idle(struct ll_conn *conn, struct proc_ctx *ctx, uint8_t evt, void *param)
 {
 	switch (evt) {
 	case LP_CC_EVT_RUN:
 		switch (ctx->proc) {
 		case PROC_CIS_CREATE:
-			lp_cc_offset_calc_req(conn, ctx, evt, param);
+			/* In case feature exchange completed after CIS create was enqueued
+			 * peer CIS peripheral support should be confirmed
+			 */
+			if (feature_peer_iso_peripheral(conn)) {
```

Comment:
this call seems to break things since feature_peer_iso_peripheral checks if LL_FEAT_BIT_CIS_PERIPHERAL is set.
LL_FEAT_BIT_CIS_PERIPHERAL is set to zero if the controller does not support cis peripheral.
in my case my controller only supports cis central.

the actual issue lies in the feature check function. they should use the BIT64(BT_LE...) defines instead.

Reply:
Correct - should be fixed by: https://github.com/zephyrproject-rtos/zephyr/pull/57862

Reply:
thanks for the quick reply. that might solve other issues as well.